management/storageservice/client.go

```diff
@@ -84,7 +84,7 @@ func (self StorageServiceClient) GetStorageServiceByLocation(location string) (*
 	return nil, nil
 }
 
-func (self StorageServiceClient) CreateStorageService(name, location string) (*StorageService, error) {
+func (self StorageServiceClient) CreateStorageService(name, location string, accountType string) (*StorageService, error) {
```

Comment:
this method ideally should accept `StorageServiceDeployment` object, that's what I meant with my initial comment.